Libraries/LibWeb/DOM/Element.cpp

```diff
@@ -2898,6 +2902,109 @@ bool Element::has_paint_containment() const
     return false;
 }
 
+int Element::number_of_owned_list_items() const
+{
+    auto number_of_owned_li_elements = 0;
+    for_each_child_of_type<DOM::Element>([&](auto& child) {
+        if (child.list_owner() == this) {
+            number_of_owned_li_elements++;
+        }
+        return IterationDecision::Continue;
+    });
+    return number_of_owned_li_elements;
+}
+
+// https://html.spec.whatwg.org/#list-owner
```

Comment:
prefer multipage links. in this case: https://html.spec.whatwg.org/multipage/grouping-content.html#list-owner

The single-page version of the spec takes quite a while to load.